src/idiom235ex.fz

```diff
@@ -0,0 +1,13 @@
+ex235 is
+
+  s := "SGVsbG8gV29ybGQh"
+
+  data :=
+    match encodings.base64.decode_str s
```

Comment:
I would prefer:
```
str := 
    match encodings.base64.decode_str s
      arr array u8 => String.type.from_bytes arr
      err error    => err.msg
```

Reply:
According to the [idiom description](https://www.programming-idioms.org/idiom/235), the string should be decoded into a byte array.

Reply:
Ah okay, hmm. Then another suggestion:
```
   match encodings.base64.decode_str s
      arr array u8 => arr
      err error    => panic "error while decoding: {err.msg}"
```